### DISCUSSION TITLE
Quarkus - ECCN

### DISCUSSION THREAD

**Discussion by chris-mulligan** [quarkusio/quarkus#39195] [Community]
I'm been asked to assess several Quarkus libraries we are using from an export control point of view and was wondering if anyone else has had to do similar or whether there are any documented ECCNs out there?

**Comment by maxandersen** [comment]
hi @chris-mulligan, IANAL but my understanding is that opensource projects that are publically available are not subject to EAR and/or  export restrictions. 

The only caveat is if the project creates or uses non-standard encryption, which, as far as I'm aware, we do not.

Vendors or redistributors of opensource projects (like i.e. Red Hat - my employer for full transparency) are under some conditions required to provide documentation for their binary distributions (i.e. Red Hat Build of Quarkus) and thus better to contact Red Hat through their support portal for that software if that is what you use or really require ECCN/EAR documentation.

**Reply by chris-mulligan** [reply]
Thanks @maxandersen, really appreciate your response and that largely tallies with my understanding. I have already logged a call with Red Hat to see how they classify the Red Hat Build of Quarkus so will see wha they say and report back.
